corehq/couchapps/__init__.py

```diff
@@ -7,8 +7,16 @@
     'users_extra': (settings.USERS_GROUPS_DB, settings.NEW_USERS_GROUPS_DB),
     'all_docs': (
         None, settings.NEW_USERS_GROUPS_DB, settings.NEW_FIXTURES_DB, 'meta',
-        settings.NEW_DOMAINS_DB),
+        settings.NEW_DOMAINS_DB, settings.NEW_APPS_DB),
     'by_domain_doc_type_date': (
         None, settings.NEW_USERS_GROUPS_DB, settings.NEW_FIXTURES_DB, 'meta',
-        settings.NEW_DOMAINS_DB),
+        settings.NEW_DOMAINS_DB, settings.NEW_APPS_DB),
+
+    # register these views to both the main db AND the apps db.
+    'app_translations_by_popularity': (None, settings.NEW_APPS_DB),
+    'exports_forms_by_app': (None, settings.NEW_APPS_DB),
+    'forms_by_app_info': (None, settings.NEW_APPS_DB),
+
+    # register just to the new apps db
+    'cloudcare_apps': settings.NEW_APPS_DB,
```

Comment:
This is a little unorthodox - I'm copying an existing view here, then registering it only to the new db.  Then during the flip, we can switch `"cloudcare/cloudcare_apps"` to `"cloudcare_apps/view"`.  After the switch we can delete the cloudcare design doc (that's the only view).  This avoids reindexing the view in the main db, although we have other options.  Let's discuss tomorrow.